sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -1161,8 +1161,21 @@ pi_result cuda_piDeviceGetInfo(pi_device device, pi_device_info param_name,
                    pi_uint64{max_alloc});
   }
   case PI_DEVICE_INFO_IMAGE_SUPPORT: {
+    pi_bool enabled;
+
+    if (std::getenv("PI_CUDA_ENABLE_IMAGE_SUPPORT") != nullptr) {
```

Comment:
@smaslov-intel, shouldn't we follow environment variables name pattern we use for [Level Zero plug-in](https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md#debugging-variables-for-level-zero-plugin) (i.e. `SYCL_PI_<Backend>_<var_name>`)?
```suggestion
    if (std::getenv("SYCL_PI_CUDA_ENABLE_IMAGE_SUPPORT") != nullptr) {
```
If we agree on that, please, update the string literal below as well.

I think we should put some documentation for this variable to https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md. @pgorlani, what do you think?

Reply:
Changing the variable name is fine for me.

Sure, I will document the variable in ``EnvironmentVariables.md``. In this regard, I will add a _Debugging variables for CUDA Plugin_ section since the images are an experimental features.

Reply:
Thanks!